packages/climate-ref-ilamb/pyproject.toml

```diff
@@ -18,14 +18,11 @@ classifiers = [
     "Topic :: Scientific/Engineering",
 ]
 dependencies = [
-    "climate-ref-core",
-    "ilamb3",
+    "climate_ref_core",
```

Comment:
`climate-ref-core` is correct. Python is agnostic about `-` vs `_`, but the dependencies should be the distribution name not necessarily the package name